### PR TITLE
Update to 7.2. Superficially tested on cloyne.org

### PR DESCRIPTION
**Pull request by clonm**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -8,19 +8,10 @@ ENV PHPMYADMIN_DBPORT 3306
 # a special database for phpMyAdmin for advanced features.
 
 RUN apt-get update -q -q && \
- apt-get install phpmyadmin --yes --force-yes && \
+ apt-get install phpmyadmin php7.2-mbstring --yes && \
```
Comment (mitar) [inline Dockerfile]: Move this `php` requirement to php image.

Also please do not remove `--force-yes` for now. It is easier to see diff. It is just a warning.

File: Dockerfile [hunk 2]
```diff
@@ -8,19 +8,10 @@ ENV PHPMYADMIN_DBPORT 3306
...
-
-COPY ./patches patches
-
-RUN \
- apt-get install patch --yes --force-yes && \
- for patch in patches/*; do patch --prefix=./patches/ -p0 --force "--input=$patch" || exit 1; done && \
- rm -rf patches && \
- apt-get purge patch --yes --force-yes && \
- apt-get autoremove --yes --force-yes
+ for file in /etc/php/7.2/mods-available/*.ini; do phpenmod $(basename -s .ini "$file"); done
```
Comment (mitar) [inline Dockerfile:15]: Remind me, why is this necessary?

File: Dockerfile (hunk 1 above)
Comment (clonm) [reply Dockerfile]: ok. should I rebase and clobber the commit where I did this, or just do a new commit reverting it?

File: Dockerfile (hunk 2 above)
Comment (clonm) [reply Dockerfile:15]: Removed the patch because it didn't apply. For line 15, I don't know whether it's necessary - I just changed the previous line 15 (which referred to php5) to get it to work.

File: Dockerfile (hunk 1 above)
Comment (mitar) [reply Dockerfile]: I do not care too much about that.

File: Dockerfile (hunk 2 above)
Comment (mitar) [reply Dockerfile:15]: Path was integrated now.

**Comment by mitar** [comment]
There was a new version of this Docker image with latest version.
